controllers/pipelines/experiment_state_handler.go

```diff
@@ -226,16 +226,16 @@ func (st ExperimentStateHandler) onUpdating(ctx context.Context, experiment *pip
 			return From(experiment.Status).WithSynchronizationState(apis.Failed).WithMessage(failureMessage)
 		}
 
-		idResult, _ := getWorkflowOutput(succeeded, ExperimentWorkflowConstants.ExperimentIdParameterName)
+		result, _ := getWorkflowOutput(succeeded, WorkflowConstants.ProviderOutputParameterName)
```

Comment:
If `providerError` is not empty it shoudl fail.

Reply:
this has now been addressed